pkg/cloud/gcp/assets/gcp-cloud-controller-manager-clusterrolebinding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: gcp-cloud-controller-manager:gcp-cloud-controller-manager
```

I think this is typically named after the service account, so maybe this should be
```suggestion
  name: gcp-cloud-controller-manager:cloud-provider
```